src/radio.cpp

```diff
@@ -0,0 +1,45 @@
+#include "radio.hpp"
+
+namespace bzzz
+{   
+    int data[16];
```

Comment:
Add documentation to `data` and perhaps rename it to something that will suggest that these are signals from the RC